specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/artifact.json

```diff
@@ -0,0 +1,1799 @@
+            "description": "If set to true. The delete cannot be revert at later time.",
+            "required": false,
+            "type": "boolean",
+            "default": false
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The batch delete of Artifact metadata is completed successfully."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/artifact/v2.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/artifacts/{origin}/{container}/batch": {
+      "delete": {
+        "tags": [
+          "Artifact"
+        ],
+        "summary": "Delete Artifact Metadata.",
+        "description": "Delete Artifact Metadata in a specific container.",
+        "operationId": "Artifacts_DeleteMetaDataInContainer",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "description": "The Azure subscription Id.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of resource group in which the workspace is located.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "workspaceName",
+            "in": "path",
+            "description": "The name of the workspace.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "origin",
+            "in": "path",
+            "description": "The origin of the Artifact.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "container",
+            "in": "path",
+            "description": "The container name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "hardDelete",
+            "in": "query",
+            "description": "If set to true. The delete cannot be revert at later time.",
+            "required": false,
+            "type": "boolean",
+            "default": false
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Artifact metadata deleted successfully."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Artifact": {
```

Comment:
According to this, `{}` is a valid Artifact instance since there is not a single required property. Is this really correct?

Reply:
fix on the way

Reply:
fixed.